zcash_client_sqlite/src/lib.rs

```diff
@@ -1087,77 +1087,52 @@ extern crate assert_matches;
 mod tests {
     use zcash_client_backend::data_api::{WalletRead, WalletWrite};
 
-    use crate::{
-        testing::{init_test_accounts_table_ufvk, network},
-        wallet::init::init_wallet_db,
-        AccountId, WalletDb,
-    };
+    use crate::{testing::TestBuilder, AccountId};
 
     #[cfg(feature = "unstable")]
-    use zcash_primitives::{
-        block::BlockHash,
-        consensus::{BlockHeight, Parameters},
-        transaction::components::Amount,
-    };
+    use zcash_primitives::{consensus::Parameters, transaction::components::Amount};
 
     #[cfg(feature = "unstable")]
     use zcash_client_backend::keys::sapling;
 
     #[cfg(feature = "unstable")]
-    use crate::{
-        chain::init::init_blockmeta_db,
-        testing::{fake_compact_block, store_in_fsblockdb, AddressType},
-        FsBlockDb,
-    };
-
-    #[cfg(feature = "unstable")]
-    use super::BlockDb;
+    use crate::testing::AddressType;
 
     #[test]
     pub(crate) fn get_next_available_address() {
-        use tempfile::NamedTempFile;
-
-        let data_file = NamedTempFile::new().unwrap();
-        let mut db_data = WalletDb::for_path(data_file.path(), network()).unwrap();
+        let mut test = TestBuilder::new().with_test_account().build();
```

Comment:
If my `TestContext` suggestion is accepted, I'd like these variables to be named `ctx`.

Reply:
Going with `st` following the `TestState` rename.